docs/documentation/concepts/things.md

```diff
@@ -47,6 +47,8 @@ A status is detailed further with a status detail object. The following table li
 |----------------------------|-------------| 
```

Comment:
Could you possibly also update the "Status Transition" diagram? This would look so much more logical, if the positions of "INITIALIZING" and "UNINITIALIZED" would be swapped...